music21/metadata/primitives.py

```diff
@@ -801,6 +803,9 @@ def __eq__(self, other) -> bool:
         >>> t2 = metadata.Text('some text', encodingScheme='scheme42')
         >>> t1 == t2
         False
+
+        Comparison with non-Text types will always be considered unequal.
+
         >>> t1 == 3
```

Comment:
better compared with `t1 == 'some text'` since that's something we might think would be equal (is a str a `non-Text` type?)